vidpub/__main__.py

```diff
@@ -96,39 +97,80 @@ def get_match_ratio(session: Session, path: pathlib.Path) -> float:
     return fuzzywuzzy.fuzz.ratio(session.title, path.stem)
 
 
-def choose_video(session: Session) -> pathlib.Path:
+def choose_video(session: Session, video_paths: list) -> pathlib.Path:
     """Look through the file list and choose the one that "looks most like it".
     """
-    score, match = max((get_match_ratio(session, p), p) for p in VIDEO_PATHS)
+    score, match = max((get_match_ratio(session, p), p) for p in video_paths)
     if score < 70:
         raise ValueError("no match")
     return match
 
 
-def build_youtube():
-    from apiclient.discovery import build
-    from google_auth_oauthlib.flow import InstalledAppFlow
+def compute_date(first_date: datetime.date, delta_days=0):
+    conference_date = first_date + datetime.timedelta(days=delta_days)
 
-    flow = InstalledAppFlow.from_client_secrets_file(
-        os.environ["OAUTH2_CLIENT_SECRET"], scopes=[YOUTUBE_UPLOAD_SCOPE]
-    )
-    credentials = flow.run_console()
-    return build("youtube", "v3", credentials=credentials)
+    return str(conference_date)
```

Comment:
I guess this function is not necessary to be separated from `extract_info` function since `compute_date` is only used in `extract_info` and `compute_data` only make simple operations.

Reply:
Done!